### PR TITLE
Headings level prediction

### PR DESCRIPTION
**Pull request by mndoye**
Prediction of the levels now done by calling directly the Python script instead of transpiling the model to JavaScript with sklearn-porter.
Note that the naming of this branch is not appropriate since we are not using tensorflow.js anymore.

### CODE REVIEW

**Comment by marianorodriguez** [comment]
Headings accuracy is lower with this changes

Before:
```json
Headings
- Identical heading detected: 989 / 1876 (52.72)%
- Level accuracy: 989 / 1440 (68.68)%
- Detected heading (wrong lvl): 1440 / 1876 (76.76)%
- False Positive: 495
- Missed Heading: 436
- Accuracy: 26.33%
- Recall: 0.7653277467764944
- Precision: 0.7836552178908249
- F1-score: 0.7629961391294625
```
After:
```json
Headings
- Identical heading detected: 988 / 1876 (52.67)%
- Level accuracy: 988 / 1420 (69.58)%
- Detected heading (wrong lvl): 1420 / 1876 (75.69)%
- False Positive: 479
- Missed Heading: 456
- Accuracy: 27.13%
- Recall: 0.7526716726313073
- Precision: 0.7782328484083015
- F1-score: 0.7494681750263226
```

**Comment by mndoye** [comment]
We don't have the same results.
Latest benchmark run:
benchmark

**Comment by marianorodriguez** [comment]
new score with updated model:

```json
Headings
- Identical heading detected: 1153 / 1876 (61.46)%
- Level accuracy: 1153 / 1523 (75.71)%
- Detected heading (wrong lvl): 1523 / 1876 (81.18)%
- False Positive: 418
- Missed Heading: 353
- Accuracy: 39.18%
- Recall: 0.7513704570659535
- Precision: 0.7752197086358633
- F1-score: 0.7468875892704004
```
